xml/art_modules.xml

```diff
@@ -674,9 +674,33 @@
   <sect2 xml:id="art-modules-python-3">
    <title>Python 3</title>
    <para>
-    This module contains the most recent version of selected Python 3 packages.
-    It has a different lifecycle than &sle; itself. Packages in this module are
-    frequently updated to new upstream versions.
+    &productname; 15 comes with a complete stack of Python 3.6 (interpreter,
+    setuptool, wheel, pip plus almost 550 additional modules). Python 3.6 is
+    fully supported by SUSE throught the complete lifecycle of SLE 15.
+   </para>
+   <para>
+    However, there are only very few pip installable modules that are still
+    compatible with 3.6 or older. Therefore, starting with SLE 15 SP4, &suse;
+    introduced the new Python 3 Module. It includes an additional modern Python
+    interpreter (plus setuptool, wheel, pip and pypi support, but no additional
+    modules). This new allows more flexibility in providing a fully supported
+    more recent Python interpreter in &productname;.
+   </para>
```

Comment:
suggest to delete all of that. the release notes explain that already and it doesn't make sense to explain in the paragraph about the python3 module something that is not in the python3 module.